README.md

```diff
@@ -296,6 +296,12 @@ spec:
         - --node=""
 ```
 
+| KSM --node parameter | FieldSelector | Note |
+| ---                  | --- | --- |
+| --node="node1"       | kubectl get pods --field-selector spec.nodeName="node1" | Fetch all pods running on node1 |
+| --node=""            | kubectl get pods --field-selector spec.nodeName= | Fetch all pods without an assigned node |
```

Comment:
Similar to my comment in https://github.com/kubernetes/kube-state-metrics/pull/2373#issuecomment-2066910515, I don't think that specifying `--node=""` is very user friendly.

We could either implicitly do it, or put it in a dedicated flag such as `--fetch-unscheduled-pods`

Reply:
Fixed the documentation to `--node=` instead of `--node=""`. Will it be clearer? Or we prefer `--fetch-unscheduled-pods`